sql/adhoc.sql

```diff
@@ -164,3 +164,33 @@ SELECT DISTINCT type, status FROM paypal ORDER BY type, status;
 SELECT type, status, SUM(amount) FROM paypal
   WHERE email='x'
   GROUP BY type, status;
+
+
+
+-- Daily totals
+
+SELECT
+  day,
+  SUM(total)
+ FROM
+   (SELECT
+        SUM(settle_amount)::numeric as total,
+        DATE(timestamp) as day
+      FROM paypal
+      WHERE timestamp > '2015-11-01'
```

Comment:
For comparison with previous years, I only need to work from 1 Nov. https://docs.google.com/spreadsheets/d/1oeaq0RAdbZE2DjjB7GTyQK9o-Us-CZaP3dWoarYFV04/edit#gid=0

I just wanted to save the query in a shared location so you'd know how I was getting to my summary amounts.